contracts/sol6/KyberStorage.sol

```diff
@@ -375,16 +375,23 @@ contract KyberStorage is IKyberStorage, PermissionGroupsNoModifiers {
         reserveIds = reservesPerTokenSrc[token];
     }
 
-    function getReserveAddressesPerTokenSrc(address token)
```

Comment:
can u add some @dev comment so its clear why we have start and end index.

Reply:
Added dev comment, please help to check if it is fit